core/src/components/toggle/test/toggle.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { newSpecPage } from '@stencil/core/testing';
+
+import { config } from '../../../global/config';
+import { Toggle } from '../toggle';
+
+describe('toggle', () => {
+  beforeEach(() => {
+    config.reset({});
+  });
+
+  const newToggle = async (): Promise<Toggle> => {
+    const { rootInstance } = await newSpecPage({
+      components: [Toggle],
+      html: `<ion-toggle></ion-toggle>`,
+    });
+    return rootInstance;
+  };
+
+  describe('enableOnOffLabels', () => {
+    it('set custom attribute on the instance, override config', async () => {
```

Comment:
Can we make these tests a bit more descriptive of the behavior we are testing? Feel free to use the suggestions below or come up with other ones.

```suggestion
    it('should disable on/off labels when setting to false on component', async () => {
```